### PR TITLE
create client table

### PR DESCRIPTION
**Pull request by dcorry586**
create client table.

### CODE REVIEW

**Review by shaunganley — APPROVED** [review]
looks good, can you include some insert and select statements to prove the feature works

**Review by shaunganley — CHANGES_REQUESTED** [review]
looks good, can you include some insert and select statements to prove the feature works

File: 003-create-client-table.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+USE ThePrimaryKeys_DylanC;
+
+-- Create client table
+CREATE TABLE IF NOT EXISTS client(
+	client_id SMALLINT UNSIGNED AUTO_INCREMENT NOT NULL,
+	name VARCHAR(255) NOT NULL,
+	address VARCHAR(255) NOT NULL,
+	phone_number VARCHAR(11) NOT NULL,
+	PRIMARY KEY (client_id),
+    sales_employee_id SMALLINT UNSIGNED NOT NULL,
```
Comment (glenn-hamilton) [inline 003-create-client-table.sql:11]: Is this needed for the feature you are currently working on?

File: 003-create-client-table.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    FOREIGN KEY (sales_employee_id) REFERENCES sales_employee(employee_id)
+);
+
+-- Insert data into table
+INSERT INTO client (name, address, phone_number, sales_employee_id)
+VALUES ('Bob Dylan', '123 somewhere over the rainbow St', '07935754333', 1);
+
+-- Select a record from table
+SELECT * FROM sales_employee;
```
Comment (glenn-hamilton) [inline 003-create-client-table.sql]: Can you be more specific with the `SELECT`

File: 003-create-client-table.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
+-- Use the team's database.
+USE ThePrimaryKeys_DylanC;
+
+-- Create client table
+CREATE TABLE IF NOT EXISTS client(
+	client_id SMALLINT UNSIGNED AUTO_INCREMENT NOT NULL,
```
Comment (shaunganley) [inline 003-create-client-table.sql:6]: this needs to be a primary key

**Review by shaunganley — CHANGES_REQUESTED** [review]

**Review by shaunganley — APPROVED** [review]

**Pull request merged**